index.md

```diff
@@ -6,13 +6,18 @@ is_home: true
 
 <p class="lead">Web sites are made of lots of things — frameworks, libraries, assets, and utilities. Bower manages all these things for you.</p>
 
+Keeping track of all these packages and making sure they are up to date (or set to the specific versions you need) is tricky. Bower to the rescue!
+
+Bower isn't limited to JavaScript libraries and can manage any front-end components, including CSS (e.g. Bootstrap), HTML or static assets like images. Bower doesn't concatenate or minify code or do anything else. All Bower does is install the right versions of the packages you need and their dependencies. It's completely dedicated to managing your packages.
+
 To [get started](#getting-started), Bower works by fetching and installing [packages](/search) from all over, taking care of hunting, finding, downloading, and saving the stuff you're looking for. Bower keeps track of these packages in a manifest file, [`bower.json`](/docs/creating-packages/#bowerjson). How you use [packages](/search) is up to you. Bower provides hooks to facilitate using packages in your [tools and workflows](/docs/tools).
 
-Bower is optimized for the front-end. Bower uses a flat dependency tree, requiring only one version for each package, reducing page load to a minimum.
+Bower is optimized for the front-end. It will install packages only once (e.g. JQuery would always be in 
```

Comment:
> Bower is optimized for the front-end. It will install packages only once (e.g. JQuery would always be in 
+`bower_components/jquery`, no matter how many packages depend on it).

Maybe: 

If a package (lets use jQuery as an example) is depended on by multiple packages, Bower will download jQuery one time only. This is known as a flat dependency graph. (In this case, NPM would download multiple versions of jQuery).

¯\(°_o)/¯. Clumsy as well.